invenio_rdm_records/services/access/service.py

```diff
@@ -104,6 +104,16 @@ def expandable_fields(self):
             GrantSubjectExpandableField("subject"),
         ]
 
+    #
+    # Update parent request on access changes
+    #
+
+    def _update_parent_request(self, parent, uow):
```

Comment:
just for my information to understand how this works:  why is it needed?

Reply:
It is needed so when a user changes the access in a record via the share button, the record/request is shown to the user that the record was shared with under their dashboard.